extra/startup.sh

```diff
@@ -89,15 +89,14 @@ _() {
         [[ -z "${TSA_PUBLIC_KEY}" ]] && echo "error: TSA_PUBLIC_KEY envvar must be set." >&2
         [[ -z "${WORKER_KEY}" ]] && echo "error: WORKER_KEY envvar must be set." >&2
         [[ -z "${TEAM_ID}" ]] && echo "error: TEAM_ID envvar must be set." >&2
+        [[ -z "${PROJECT}" ]] && echo "error: PROJECT envvar must be set." >&2
```

Comment:
Not sure about making it required.
I mean mostly for current CF stack which are already created, if you merge this PR you might broke the already running workers because PROJECT is a new parameter and require to update the CF stack.

Reply:
True, but not the only breaking change as `ENV` and `ROLE` are also required.

So I could make it backwards compatible or update the worker of the customer. How many of them are using the new `curl | bash` setup ?